### PR TITLE
Add support for SPA sites with custom_error_response

### PR DESCRIPTION
**Pull request by enjoy2000**
- My branch is based on https://github.com/cloudmaniac/terraform-aws-static-website/pull/10 as subdomain is my use case
- Add support for SPA sites with custom_error_response
- Ran and tested on my infra

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -295,6 +295,14 @@ resource "aws_cloudfront_distribution" "website_cdn_redirect" {
     bucket = aws_s3_bucket.website_logs.bucket_domain_name
     prefix = "${var.website-domain-redirect}/"
   }
+  dynamic "custom_error_response" {
```
Comment (cloudmaniac) [inline main.tf]: @hat-diag The Terraform module builds 2 CloudFront Distribution: `website_cdn_redirect` and `website_cdn_root`. You added this block only to the `website_cdn_redirect` distribution but I assume it's also required for the main one?

**Comment by cloudmaniac** [comment]
Up. @enjoy2000 Kind follow-up on my [query](https://github.com/cloudmaniac/terraform-aws-static-website/pull/11#discussion_r808358037) above.

File: main.tf (hunk 1 above)
Comment (enjoy2000) [reply main.tf]: @cloudmaniac sorry for the late response. Yes, it's actually my bad I put it in the wrong distribution, we won't need it for `website_cdn_redirect` as it needs to always return redirect response.
Let me update 🙏 .

**Comment by enjoy2000** [comment]
@cloudmaniac Please help to check again

**Review by cloudmaniac — COMMENTED** [review]
General review done

**Pull request merged**